src/containers/cyclicbuffer.d

```diff
@@ -0,0 +1,828 @@
+/**
+ * Cyclic Buffer
+ * Copyright: © 2016 Economic Modeling Specialists, Intl.
```

Comment:
I'm not sure how EMSI gets the copyright on something it didn't write. @jwhear thoughts?